analyzer/pkg/rpm/rpm.go

```diff
@@ -24,6 +26,7 @@ var requiredFiles = []string{
 	"usr/lib/sysimage/rpm/Packages",
 	"var/lib/rpm/Packages",
 }
+var errUnexpectedNameFormat = errors.New("unexpected name format")
```

Comment:
For stacktrace, we still use xerrors.

```suggestion
var errUnexpectedNameFormat = xerrors.New("unexpected name format")
```